presto-main/src/test/java/com/facebook/presto/block/TestDictionaryBlock.java

```diff
@@ -153,7 +189,12 @@ public void testCompactAllKeysReferenced()
         assertEquals(compactBlock.isCompact(), true);
     }
 
-    private static DictionaryBlock createDictionaryBlockWithUnreferencedKeys(Slice[] expectedValues, int positionCount)
+    private static DictionaryBlock createDictionaryBlock(Slice[] expectedValues, int[] ids, DictionaryId dictionaryId)
+    {
+        return new DictionaryBlock(ids.length, new SliceArrayBlock(expectedValues.length, expectedValues), wrappedIntArray(ids), dictionaryId);
+    }
+
+    private static DictionaryBlock createDictionaryBlockWithUnreferencedKeys(Slice[] expectedValues, int positionCount, DictionaryId dictionaryId)
```

Comment:
With the change to explicitly list the dictionary indexes above, you don't need any of the changes from this line down in the file.